Controllers/OrdersController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using API.Models;
+using API.Models.DTO;
+using API.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using API.Models.DTO.Administrator;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrdersController : ApiControllerBase
+    {
+
+        private readonly UserManager<Employee> _userManager;
+
+        public OrdersController(ApiContext context, UserManager<Employee> userManager) :
+            base(context) {
+            _userManager = userManager; 
+        }
+
+        [HttpGet("orders")] //return a list of orders without products
```

Comment:
We dont need to specify `orders` endpoint here because route is already `api/orders`. 
With this additional specification we get `api/orders/orders`. `[HttpGet]` is perfectly fine

Reply:
Changed